mypy/semanal.py

```diff
@@ -2518,6 +2518,8 @@ def visit_decorator(self, dec: Decorator) -> None:
                 self.check_decorated_function_is_method('classmethod', dec)
             elif (refers_to_fullname(d, 'builtins.property') or
                   refers_to_fullname(d, 'abc.abstractproperty')):
+                # TODO: Stop treating properties as special cases and treat as generalized
+                # TODO: descriptors
```

Comment:
This comment is not needed, we already have an issues for this.

Reply:
Removed